doc/source/using-as-a-library.rst

```diff
@@ -139,7 +139,7 @@ A suitable list of automagics for a particular plugin (based on operating system
 
 This will take the plugin module, extract the operating system (first level of the hierarchy) and then return just
 the automagics which apply to the operating system.  Each automagic can exclude itself from being used for specific
-operating systems, so that an automagic designed for linux is not used for windows or mac plugins.
+operating systems, so that for example an automagic designed for linux is not used for windows or mac plugins.
```

Comment:
I'm not sure this is suitably placed?  It should also be between parenthetical commas really?  Perhaps `such that an automagic` rather than `so that an automagic`?